### PR TITLE
Small edits in .github/CONTRIBUTING.markdown

### PR DESCRIPTION
**Pull request by SuriyaaKudoIsc**
Adding some full stop / period.

### CODE REVIEW

**Comment by pathawks** [comment]
I don’t know that list items need a period.

**Comment by parkr** [comment]
It matches the previous line.

File: .github/CONTRIBUTING.markdown [hunk 1]
```diff
@@ -36,7 +36,7 @@ Whether you're a developer, a designer, or just a Jekyll devotee, there are lots
 
 Many small changes can be made entirely through the github.com web interface.
 
-1. Navigate to the file within [`jekyll/jekyll`](https://github.com/jekyll/jekyll) that you'd like to edit
+1. Navigate to the file within [`jekyll/jekyll`](https://github.com/jekyll/jekyll) that you'd like to edit.
 2. Click the pencil icon in the top right corner to edit the file
```
Comment (pathawks) [inline .github/CONTRIBUTING.markdown]: No period after this line?

File: .github/CONTRIBUTING.markdown [hunk 2]
```diff
@@ -48,8 +48,8 @@ That's it! You'll be automatically subscribed to receive updates as others revie
 
 ### Submitting a pull request via Git command line
 
-1. Fork the project by clicking "Fork" in the top right corner of [`jekyll/jekyll`](https://github.com/jekyll/jekyll)
-2. Clone the repository lcoally `git clone https://github.com/<you-username>/jekyll`
+1. Fork the project by clicking "Fork" in the top right corner of [`jekyll/jekyll`](https://github.com/jekyll/jekyll).
+2. Clone the repository lcoally `git clone https://github.com/<you-username>/jekyll`.
 3. Create a new, descriptively named branch to contain your change ( `git checkout -b my-awesome-feature` ).
 4. Hack away, add tests. Not necessarily in that order.
 5. Make sure everything still passes by running `script/cibuild` (see [the tests section](#running-tests-locally) below)
```
Comment (pathawks) [inline .github/CONTRIBUTING.markdown]: No period after this line?

**Pull request merged**